### DISCUSSION TITLE
Question: May you plan a stable release?

### DISCUSSION THREAD

**Discussion by MarcosCobena** [SixLabors/Fonts#147] [General] [answered]
Hi!

This is more a question than an issue but, the link for such returns 404 currently. I hope am leaving it where expected :-)

We are testing this library to render one-line fonts and, up to now, it works like a charm :-) Congratulations for the great job guys, the API is a joy to use as well.

We are developing a solution for a customer. I have seen ImageSharp is going through a RC process and, if possible, would like to ask whether you plan the same for Fonts too.

Today I have discovered you offer a Commercial License which, may be an option for us too.

Thanks in advance,

—Marcos

**Comment by MagicAndre1981** [comment]
any updates on this? After 19 betas a stable 1.0 should be ok. Other things can be done in 1.1

**Reply by Bykiev** [reply]
Is any news about it?

**Reply by JimBobSquarePants** [reply]
Dude.. give it a break. There’s a milestone with an estimated release date. Read the damn thing.

**Comment by Bykiev** [comment]
> Dude.. give it a break. There’s a milestone with an estimated release date. Read the damn thing.

But all planned tasks already finished and you've changed the milestone date recently...

**Reply by JimBobSquarePants** [reply]
Things change. If you want things to go faster contribute.

**Reply by Bykiev** [reply]
I don't know much about the fonts, but I'm ready to contribute. What help is needed?

**Reply by JimBobSquarePants** [reply]
You're the first person I've ever challenged who said "ok". Kudos! 

However, I have good news, read below.

**Comment by JimBobSquarePants** [comment, marked as answer]
Now.

https://sixlabors.com/posts/announcing-fonts-100/
